### PR TITLE
Now formatting target code to remove trailing dash.

### PR DESCRIPTION
**Pull request by bgsavage**
This is a global change for the app and will propagate all places in the app where target is displayed & used.
Added unit test.

### CODE REVIEW

File: webapp/src/main/webapp/src/app/assessments/assessment-exam.mapper.ts [hunk 1]
```diff
@@ -151,6 +151,14 @@ export class AssessmentExamMapper {
     return uiModel;
   }
 
+  formatTarget(target) {
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/app/assessments/assessment-exam.mapper.ts]: Ideally this should be private inside this mapper and is only public to make it easy to unit test.  If we have another formatting function, we should consider creating an AssessmentExamFormatter to facilitate unit testing of all the formatting functions.

File: webapp/src/main/webapp/src/app/assessments/assessment-exam.mapper.ts [hunk 2]
```diff
@@ -151,6 +151,14 @@ export class AssessmentExamMapper {
...
+    let dashIndex = target.indexOf("-");
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/assessments/assessment-exam.mapper.ts]: are all target codes two segments?
wondering if this should be lastIndexOf() or not

Comment (bgsavage) [inline webapp/src/main/webapp/src/app/assessments/assessment-exam.mapper.ts]: That's a good point, let me look and see what we have in UAT and ETS

Comment (bgsavage) [inline webapp/src/main/webapp/src/app/assessments/assessment-exam.mapper.ts]: I did queries in UAT and ETS counting the dashes and didn't find any where the number of dashes was more than one, so I think we're good here.

**Comment by bgsavage** [comment]
It does not since we are using target id to look it up.  Good thought though

**Pull request merged**